packages/gdpr-tool/package.json

```diff
@@ -0,0 +1,36 @@
+{
+    "name": "@commercetools/gdpr-tool",
+    "version": "1.0.0-development",
```

Comment:
Nitpick: I never saw this kind of version. Why not simply `1.0.0`?

Reply:
By using `1.0.0-development` it will be bumped to `1.0.0` during npm release. No matter if we choose `major`, `minor` or `patch`. 

Im not 100% sure but I think if I named it `1.0.0` here it would be bumped to `1.0.1` if I release as `patch`, for example.